edb/lib/std/25-numoperators.eql

```diff
@@ -300,12 +318,22 @@ std::`?!=` (l: OPTIONAL std::float64, r: OPTIONAL std::float64) -> std::bool
 
 
 CREATE INFIX OPERATOR
-std::`!=` (l: std::decimal, r: std::decimal) -> std::bool
+std::`!=` (l: std::decimal, r: std::anyreal) -> std::bool
+    FROM SQL OPERATOR r'<>';
+
+
+CREATE INFIX OPERATOR
+std::`?!=` (l: OPTIONAL std::decimal, r: OPTIONAL std::anyreal) -> std::bool
+    FROM SQL EXPRESSION;
```

Comment:
Please use this style consistently:

```
{
     FROM SQL $$
     SELECT round("val")
     $$;
 };
```

Reply:
We consistently omit `{...}` for one-liners like `FROM SQL EXPRESSION`.